src/lib/inputs/InputColor.svelte

```diff
@@ -0,0 +1,144 @@
+<script lang="ts">
+    import { createEventDispatcher, onMount } from "svelte";
+
+    // Props
+    export let size: "xsmall" | "small" | "medium" | "large" | "xlarge" =
+        "medium";
+    export let value: string = "#000000";
+    export let style: string = "";
+    export let label: boolean = false;
+    export let labelColor: string = "#000";
+    export let disabled: boolean = false;
+
+    // Event dispatcher
+    const dispatch = createEventDispatcher();
+
+    // Variables
+    let inputClassList = ["input", size];
+    let inputClassString = "";
+
+    // Lifecycle Hooks
+    onMount(() => {
+        inputClassString = inputClassList.join(" ");
+    });
+
+    // Reactive Statements
+    $: {
+        inputClassList = ["input", size];
+        inputClassString = inputClassList.join(" ");
+    }
+    $: inputStyle = `
+        ${style}
+    `;
+</script>
+
+<div class="container">
+    {#if label}
+        <label class="label" for={$$restProps.id} style="color: {labelColor}">
+            {$$restProps.id}
+        </label>
+    {/if}
+
+    <input
+        {value}
+        {disabled}
+        readonly={disabled}
+        {...$$restProps}
+        id={label ? $$restProps.id : undefined}
+        type="color"
+        style={inputStyle}
+        class={inputClassString}
+        on:input={(e) => dispatch("input", e)}
+        on:focus
+        on:blur
+        on:change
+        on:click
+        on:keydown
+        on:keyup
+    />
+</div>
+
+<style lang="scss">
+    .container {
+        position: relative;
+        display: inline-flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: center;
+        gap: 0.5rem;
+        padding: 0;
+        background: transparent;
+    }
+
+    input[type="color"] {
+        -webkit-appearance: none;
+        -moz-appearance: none;
+        appearance: none;
+        background-color: transparent;
+        border: none;
+        cursor: pointer;
```

Comment:
in general for inputs are you changing styling for when they're disabled, like graying out background or cursor: default or cursor: not-allowed

Reply:
The background will become #ccc and the disabled attribute will be added to the element.

Reply:
Though, on review, I'm seeing that I didn't do that with this component... xD Fixing now.